### PR TITLE
updating netcore and variables for AspNetCore.Tests

### PR DESCRIPTION
**Pull request by eddynaka**
Fix Issue #1769

### CODE REVIEW

**Comment by TimothyMothra** [comment]
Hey Eddy, can you confirm that this compiles locally?
You're getting build errors:
> Microsoft.ApplicationInsights.AspNetCore.Tests.csproj(0,0): Error NU1605: Detected package downgrade: Microsoft.Extensions.Configuration.Json from 2.1.0 to 1.0.2. Reference the package directly from the project to select a different version.

File: NETCORE/.vsts/linux-build.yml [hunk 1]
```diff
@@ -29,10 +32,21 @@ steps:
...
 
-- task: DotNetCoreInstaller@2
-  displayName: install dotnet core 2.2.104
+- task: DotNetCoreCLI@1
+  displayName: Unit Tests for AspNetCore 3.1
+  continueOnError: true
+  inputs:
+    command: "test"
+    projects: "NETCORE/test/**/*AspNetCore.Tests.csproj"
+    arguments: "--configuration Release --framework netcoreapp3.1 -l trx"
```
Comment (TimothyMothra) [inline NETCORE/.vsts/linux-build.yml:41]: these tests aren't being run. We need to run this command locally and confirm that it's working.

Comment (eddynaka) [reply NETCORE/.vsts/linux-build.yml:41]: ok! just changed to full name and it worked!

**Pull request merged**